aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -4310,6 +4310,30 @@ TORCH_IMPL_FUNC(linalg_ldl_factor_ex_out)
   }
 }
 
+std::tuple<Tensor&, Tensor&> linalg_ldl_factor_out(
+    const Tensor& self,
+    bool hermitian,
+    Tensor& LD,
+    Tensor& pivots) {
+  auto info = at::empty({0}, self.options().dtype(kInt));
+  // We pass check_errors as we want to use lu_factor rather than lu_factor_ex
+  // in the errors
+  at::linalg_ldl_factor_ex_outf(
+      self, hermitian, /*check_errors=*/false, LD, pivots, info);
+  at::_linalg_check_errors(info, "torch.linalg.ldl_factor", self.dim() == 2);
+  return std::tie(LD, pivots);
+}
+
+std::tuple<Tensor, Tensor> linalg_ldl_factor(
+    const Tensor& self,
+    bool hermitian) {
+  Tensor LD, pivots, info;
+  std::tie(LD, pivots, info) =
+      at::linalg_ldl_factor_ex(self, hermitian, /*check_errors=*/false);
```

Comment:
This should be `chech_errors=True` by default. At least that's what we do in `lu_factor`.

Reply:
It's `false` in `lu_factor`. It's done so that the error message has the function name `ldl_factor` and not `ldl_factor_ex`.

Reply:
You are completely right. Even though I wrote that function a couple months ago I totally misremembered this.